data/i18n/bn/bn.toml

```diff
@@ -325,25 +425,100 @@ other = "এই নিবন্ধটি এক বছরেরও বেশি
 other = "প্যাচ রিলিজ"
 
 [post_create_issue]
-other = "একটি ইশূ তৈরি করুন"
+other = "একটি ইস্যু তৈরি করুন"
 
 [prerequisites_heading]
 other = "শুরু করার আগে"
 
 [previous_patches]
 other = "প্যাচ রিলিজ:"
 
+# The following text is displayed when JavaScript isn't available.
+[release_binary_alternate_links]
+other  = """You can find links to download Kubernetes components (and their checksums) in the [CHANGELOG](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG) files.
+Alternately, use [downloadkubernetes.com](https://www.downloadkubernetes.com/) to filter by version and architecture."""
+
+[release_binary_arch]
+other  = "আর্কিটেকচার"
+
+[release_binary_arch_option]
+other = "আর্কিটেকচারগুলো"
+
+[release_binary_copy_link]
+other = "কপি লিংক"
+
+[release_binary_copy_link_certifcate]
+other = "cert"
+
+[release_binary_copy_link_checksum]
+other = "checksum"
+
+[release_binary_copy_link_signature]
+other = "signature"
+
+[release_binary_copy_link_tooltip]
+other = "ক্লিপবোর্ডে কপি করুন"
+
+[release_binary_download]
+other  = "বাইনারি ডাউনলোড করুন"
+
+[release_binary_download_tooltip]
+other = "বাইনারি ফাইল ডাউনলোড করুন"
+
+[release_binary_options]
+other  = "ডাউনলোড অপশন"
+
+[release_binary_os]
+other = "অপারেটিং সিস্টেম"
+
+[release_binary_os_option]
+other = "অপারেটিং সিস্টেমগুলো"
+
+[release_binary_os_darwin]
+other = "darwin"
+
+[release_binary_os_linux]
+other = "linux"
+
+[release_binary_os_windows]
+other = "windows"
+
+[release_binary_table_caption]
+other = "কুবারনেটিস কম্পোনেন্ট বাইনারি ডাউনলোড করুন"
+
+# NOTE: <current-version> is a placeholder for the actual version number set by 'release-binaries' shortcode.
+# Please do not localize or modify <current-version> placeholder.
+[release_binary_section]
+other = """You can find the links to download <current-version> Kubernetes components (along with their checksums) below.
+To access downloads for older supported versions, visit the respective documentation 
+link for [older versions](https://kubernetes.io/docs/home/supported-doc-versions/#versions-older) or use [downloadkubernetes.com](https://www.downloadkubernetes.com/).""" 
+
+# NOTE: <current-version> and <current-changelog-url> are placeholders set by 'release-binaries' shortcode.
+# Please do not localize or modify <current-version> and <current-changelog-url>  placeholders.
+[release_binary_section_note]
+other = """To download older patch versions of <current-version> Kubernetes components (and their checksums), 
+please refer to the [CHANGELOG](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG/CHANGELOG-<current-changelog-url>.md) file.""" 
```

Comment:
I'd expect to see Bangla here. You can omit this if you don't want to add a translation yet.